python-frontend/typeshed_serializer/build-with-docker.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+export USER_ID="$(id -u)"
+export GROUP_ID="$(id -g)"
+docker compose -f ./docker/docker-compose.yml --project-directory ./docker up
```

Comment:
Does docker compose go "down" automatically after finishing the serialization ?

Reply:
yes, the container will stop